### ISSUE TITLE
Was machen Zombies nach Treffen des Spielers ?

### ISSUE THREAD

**Issue by storm85049** [storm85049/beer_the_zombie#1]
Laufen die Zombies alle sie ganze Zeit konstant weiter oder hält einer an wenn man in ihn reinläuft ? Das würde mehr Sinn machen. Schreibt mal was ihr darüber denkt

**Comment by JayInteger** [comment]
Ja, er sollte aufjedenfall eine Sekunde stehen bleiben und genau, dass selbe sofern man einen Zombie trifft
